pyroute2/ipdb/interface.py

```diff
@@ -779,9 +779,18 @@ def error(x):
             #
             # One simple way to work that around is to remove
             # secondaries first.
-            rip = sorted(removed['ipaddr'],
-                         key=lambda x: self['ipaddr'][x]['flags'],
```

Comment:
I merged it first, but then realized, that it could be solved in another way, as
```python
key=lambda x: self['ipaddr'][x]['flags'] or 0xffff
```

What woud you say?